tools/export/exporters.py

```diff
@@ -196,6 +197,7 @@ def scan_and_copy_resources(self, prj_paths, trg_path, relative=False):
             self.config_header = self.toolchain.MBED_CONFIG_FILE_NAME
             config.get_config_data_header(join(trg_path, self.config_header))
             self.config_macros = []
+            self.resources.inc_dirs.append(dirname(self.config_header))
```

Comment:
Actually that does some strange stuff. `-I..//home/jimbri01/src/mbedmicro/mbed/.build/export/.temp` ends up in my command line.